src/app/header/components/search-bar/search-bar.component.scss

```diff
@@ -1,16 +1,18 @@
+$greenWalla: #13C1AC;
```

Comment:
It was more about having the variable for all the project as we usually use the same color palette across all the project.

For example you are using `#13C1AC` in SearchBar, Selector and Slider.

Reply:
Yeah, that makes sense, would it be better to create a branch to do that change in the entire application?

Reply:
Do it here in this case, don't worry.